forge.config.ts

```diff
@@ -55,6 +55,7 @@ const config: ForgeConfig = {
 			? []
 			: [
 					new MakerDMG(
+						// @ts-expect-error - https://github.com/electron/forge/issues/3712
```

Comment:
There is bug in the MakerDMG - `appPath` is required by mistake, and it should be optional in the next release. So we will remove this `@ts-expect-error` when it's fixed.

Reply:
The bug is closed. Does it mean it was not released yet?

Reply:
They made it [here](https://github.com/electron/forge/pull/3761/files) optional, but looks like it wasn't in 7.6.0 release (according to [this](https://github.com/electron/forge/issues/3712#issuecomment-2508775834) comment)